src/lib/tracking.ts

```diff
@@ -37,8 +38,21 @@ export const addTrackingParams = (
     };
 
     const queryString = Object.entries(trackingLinkParams).map(([key, value]) => `${key}=${value}`);
+    const alreadyHasQueryString = baseUrl.includes('?');
 
-    return `${baseUrl}?${queryString.join('&')}`;
+    return `${baseUrl}${alreadyHasQueryString ? '&' : '?'}${queryString.join('&')}`;
+};
+
+export const addRegionIdAndTrackingParamsToSupportUrl = (
+    baseUrl: string,
+    tracking: EpicTracking | BannerTracking,
+    countryCode?: string,
+): string => {
+    const isSupportUrl = baseUrl.search(/(support.theguardian.com)\/(contribute|subscribe)/) >= 0;
```

Comment:
This possibly won't work on CODE. I'd just test that the domain contains `support.`

`const isSupportUrl = /\bsupport\./.test(baseUrl);`

Reply:
yeah makes sense  👍